lang/python/Jinja2/Makefile

```diff
@@ -25,7 +25,7 @@ define Package/python3-jinja2
   CATEGORY:=Languages
   SUBMENU:=Python
   TITLE:=Jinja2
-  URL:=http://jinja.pocoo.org/
+  URL:=https://jinja.palletsprojects.com/en/2.11.x/
```

Comment:
I'd say https://jinja.palletsprojects.com/ is a better option here, even if it's currently redirecting to the versioned sub tree.  or even https://palletsprojects.com/p/jinja/ which links "jinja.palletsproject.com" as the url for docs

Reply:
Thanks @karlp for review. I was not quite sure about it here.
Like you are saying there is an versioned subtree, but there could be released version 2.12 and on the router, there is installed 2.11 and it woudn't match. 

The website I was thinking for that and skipped that. Don't know the reason now, but maybe that the previous URL pointed to doc.

Anyway, I don't mind putting there https://palletsprojects.com/p/jinja/ .
Is that OK for you?

Reply:
works for me, I just meant that in general we link to the _project_ page, not a particular version.

Reply:
Changed. If you agree, please resolve the discussion and I will merge it.

Reply:
oh, sorry, I had only meant to comment, not "start a review" :)